cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -1392,7 +1399,13 @@ private enum CruiseControlIssues {
         /**
          * Cruise Control was not declared in the Kafka Resource
          */
-        cruiseControlDisabled("Cruise Control disabled");
+        cruiseControlDisabled("Cruise Control disabled"),
+
+        /**
+         * Kafka Cluster is not ready
+         *
+         */
+        kafkaClusterNotReady("Creating Kafka Cluster");
```

Comment:
If it is named` kafkaClusterNotReady` maybe it should say `Kafka cluster is not ready` and not `Creating Kafka Cluster`?